src/main/scala/com/adobe/api/platform/runtime/mesos/package.scala

```diff
@@ -168,7 +168,7 @@ package object mesos {
 
   object DefaultTaskBuilder extends TaskBuilder {
 
-    def apply(reqs: TaskDef, offer: Offer, resources: Seq[Resource], portMappings: Seq[PortMapping])(
+    def apply(reqs: TaskDef, offer: Offer, resources: Seq[Resource], portMappings: Seq[PortMapping], commandDef: CommandDef = null)(
```

Comment:
to avoid null check below, just set a default here: 
``` commandDef: CommandDef = new CommandDef()```

or is it even possible to NOT specify a commandDef? (no default value, and no null check)

(setting or using null at all is typically not a scalaism; if it is truly optional make it Option[CommandDef] = None)